plugins/CircuitExplorer/api/neuron/NeuronGeometryType.h

```diff
@@ -24,5 +24,6 @@ enum class NeuronGeometryType
 {
     Original,
     Smooth,
```

Comment:
Smooth is RadiusSmooth right ? Maybe we can rename it (if yes also in the Python API).